osmaxx-py/osmaxx/excerptexport/forms/excerpt_order_form_helpers.py

```diff
@@ -23,10 +24,16 @@ def other_public(user):
     return _get_active_excerpts().filter(is_public=True).exclude(owner=user)
 
 
-def countries():
-    return Excerpt.objects.filter(
-        bounding_geometry__osmosispolygonfilterboundinggeometry__isnull=False
-    )
+def get_prefixed_countries():
+    return get_authenticated_api_client().get_prefixed_countries()
+
+
+def get_country_choices():
+    return tuple((country['id'], country['name']) for country in get_prefixed_countries())
+
+
+def get_country_id_to_name_dict():
+    return {country['id']: country['name'] for country in get_authenticated_api_client().get_country_list()}
```

Comment:
The only place where this is used, is https://github.com/geometalab/osmaxx/pull/299/files#diff-2f2fab63c402770e035d997760c4aa42R71

There, only a single value is mapped with the dict. So instead of a helper method returning the dict, we could offer a helper method that does maps a single country ID to the corresponding country name.